webrtc.html

```diff
@@ -4068,10 +4068,13 @@ <h2 id="sec.cert-mgmt">Certificate Management</h2>
           </dd>
 
           <dt>sequence&lt;ArrayBuffer&gt; getRemoteCertificates()</dt>
-
+          
           <dd>
-            <p>Returns the certificate chain in use by the remote side, with each certificate
-            encoded in binary Distinguished Encoding Rules (DER) [[!X690]].</p>
+            <p>Returns the certificate chain in use by the remote side, with each certificate encoded
+            in binary Distinguished Encoding Rules (DER) [[!X690]]. If the remote side offers multiple
+            certificates, then <code>getRemoteCertificates()</code> will return an empty list prior to
+            selection of the remote certificate, which will be completed by the time
+            <code><a>RTCDtlsTransportState</a></code> transitions to "connected".</p>
```

Comment:
If it's in "new"  (before the remote certificates are presented), what will be returned? (Suggest empty list)
The text seems to only return empty when multiple certificates are offered, which doesn't make sense for the "new" state.